<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ROLES TESTCASES"
           tests="7"
           failures="1"
           errors="0"
           skipped="0"
           time="296.423">
  <properties />
            <testcase name="create_a_new_role" time="66.530" classname="access.Roles_TestCases">
                <failure type="java.lang.AssertionError"
                  message="expected [CODA - Roles] but found [CODA - Roles - Add]"
              >
<![CDATA[java.lang.AssertionError: expected [CODA - Roles] but found [CODA - Roles - Add]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:513)
  org.testng.Assert.assertEqualsImpl(Assert.java:135)
  org.testng.Assert.assertEquals(Assert.java:116)
  org.testng.Assert.assertEquals(Assert.java:190)
  org.testng.Assert.assertEquals(Assert.java:200)
  access.RolesAddPage.Submit(RolesAddPage.java:105)
  access.Roles_TestCases.create_a_new_role(Roles_TestCases.java:23)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:756)
  org.testng.TestRunner.run(TestRunner.java:610)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="fail_to_create_role_when_require_field_is_not_populated" time="44.378" classname="access.Roles_TestCases" />
                <testcase name="delete_role_from_database" time="2.272" classname="access.Roles_TestCases" />
                <testcase name="create_a_new_role_without_a_name" time="53.159" classname="access.Roles_TestCases" />
                <testcase name="select_cancel_button_in_roles_add_page" time="33.877" classname="access.Roles_TestCases" />
                <testcase name="acces_level_field_is_empty_in_role_add_page" time="49.670" classname="access.Roles_TestCases" />
                <testcase name="create_a_duplicate_role" time="46.537" classname="access.Roles_TestCases" />
      
</testsuite>
